packages/expo/tsconfig.base.json

```diff
@@ -3,16 +3,15 @@
   "display": "Expo",
 
   "compilerOptions": {
-    "target": "esnext",
-    "lib": ["dom", "es6", "es2016.array.include", "es2017.object"],
```

Comment:
Doing some research, this looks safe since Array.include is supported on all target platforms that Expo supports, and the [es2017.object](https://github.com/microsoft/TypeScript/blob/master/src/lib/es2017.object.d.ts) methods are supported here https://github.com/facebook/react-native/blob/dc80b2dcb52fadec6a573a9dd1824393f8c29fdc/packages/polyfills/Object.es7.js.

In the future, please include this level of research in your PR when making breaking changes.

Reply:
Will do!

What platforms is it that Expo supports? I thought the code was always running very recent versions of Node.js, so that support for the upcoming ECMAScript features were pretty much guaranteed.